src/tile/FeaturedTile.js

```diff
@@ -0,0 +1,133 @@
+import React, { PropTypes } from 'react';
+import {
+	TouchableOpacity,
+	View,
+	Image,
+	StyleSheet,
+	Dimensions,
+} from 'react-native';
+import Text from '../text/Text';
+import Icon from '../icons/Icon';
+
+const FeaturedTile = ({
+	title,
+	icon,
+	caption,
+	imageSrc,
+	onPress,
+	activeOpacity,
+	containerStyle,
+	imageContainerStyle,
+	overlayContainerStyle,
+	titleStyle,
+	captionStyle,
+	width,
+	height,
+}) => {
+	if (!width) {
+		width = Dimensions.get('window').width;
+	}
+	if (!height) {
+		height = width * 0.8;
```

Comment:
Height is always `0.8 * width`? 
1. If the tile is a square by default, shouldn't the `width = height`?
2. Since width is customizable, maybe let the user set the height too?